apps/odyssey-stats/src/app.scss

```diff
@@ -63,7 +63,7 @@
 	}
 
 	// Black segmented controls.
-	.is-section-stats .segmented-control .segmented-control__item {
+	.is-section-stats ul.segmented-control li.segmented-control__item {
```

Comment:
Fix green Segment Control on summary page by the way:

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/1425433/235545292-ecd8e07e-30db-4872-81de-bac39ebe77cf.png">